package-lock.json

```diff
@@ -1,8 +1,9482 @@
 {
   "name": "@microsoft/connected-workbooks",
   "version": "1.0.0",
-  "lockfileVersion": 1,
```

Comment:
why is this file part of the change?

Reply:
New `package-lock.json` format in npmV7, you can check the changes [here](https://blog.npmjs.org/post/621733939456933888/npm-v7-series-why-keep-package-lockjson.html)